spring-cloud-app-broker-deployer-cloudfoundry/src/test/java/org/springframework/cloud/appbroker/deployer/cloudfoundry/CloudFoundryAppDeployerTest.java

```diff
@@ -136,6 +138,7 @@
 	@BeforeEach
 	void setUp() {
 		deploymentProperties = new CloudFoundryDeploymentProperties();
+		deploymentProperties.setApiPollingTimeout(EXPECTED_COMPLETION_DURATION); //set a different value than the default to make sure it gets loaded
```

Comment:
Please, remove the comment, the name seems explicit enough.

Reply:
thanks, this is fixed with  bd25634